pkg/daemon/update.go

```diff
@@ -195,6 +198,29 @@ func (dn *Daemon) writeUnits(units []ignv2_2types.Unit) error {
 			return fmt.Errorf("Failed to write systemd unit %q: %v", u.Name, err)
 		}
 
+		// if the unit doesn't note if it should be enabled or disabled then
+		// skip all linking.
+		// if the unit should be enabled, then enable it.
+		// otherwise the unit is disabled. run disableUnit to ensure the unit is
+		// disabled. even if the unit wasn't previously enabled the result will
+		// be fine as disableUnit is idempotent.
+		// Note: we have to check for legacy unit.Enable. if it is true set the
```

Comment:
nit: this note seems to describe a flow that isn't happening anymore? since we just call `enableUnit` directly if `Enable` is true

Reply:
Woops!